plugin/util.go

```diff
@@ -16,16 +16,19 @@ import (
 // helper function to extract the issue number from
 // the commit details, including the commit message,
 // branch and pull request title.
-func extractIssue(args Args) string {
-	return regexp.MustCompile(args.Project + "\\-\\d+").FindString(
-		fmt.Sprintln(
-			args.Commit.Message,
-			args.PullRequest.Title,
-			args.Commit.Source,
-			args.Commit.Target,
-			args.Commit.Branch,
-		),
-	)
+func extractIssues(args Args) []string {
+
+    regex := regexp.MustCompile(args.Project+"-\\d+")
+    matches := regex.FindAllString(fmt.Sprintln(
+		args.AdditionalMessage,
+        args.Commit.Message,
+        args.PullRequest.Title,
+        args.Commit.Source,
+        args.Commit.Target,
+        args.Commit.Branch,
+    	), -1)
+
+    return matches
```

Comment:
```suggestion
    return removeDuplicates(matches)
```

make use of the removeDuplicate in here and not outside, so that the returned values contains the valid keys

Reply:
done